src/lacosmic.py

```diff
@@ -16,7 +16,7 @@
 # high-level functions to check if an observation is half-frame or N+S
 def is_halfframe(inimg, data_hdu=0):
     f = pyfits.open(inimg)
-    ccdsec = f[data_hdu].header['CCDSEC']
```

Comment:
Yes, you are right and this is actually something I have on my list of things to check (probably next week) so we can delete many of those defunct scripts (there are quite a few) and also those not required for the automation, and keep maintaining the others. 
For the moment, I am just making the pipeline work. That is, reducing one-night data in classical and nod-and-shuffle modes as the automated pipeline will do (e.g. we don't need to do multinight reductions).

Reply:
Ok, so long as it's getting followed up on soon no worries. I can check back on this list as we keep going. Otherwise I'd say make an issue to keep track.